telethon/client/updates.py

```diff
@@ -16,7 +16,9 @@ class UpdateMethods(UserMethods):
 
     def run_loop(self):
         """
-        Runs the event loop.
+        Runs the event loop until a disconnection occurs, either from
+        Telegram or by user action. If the loop is already running you
+        should ``await client.connection_dropped`` instead.
```

Comment:
I'd change "from Telegram" to "because of an error", and "If the loop is already running" to "In async code".

Reply:
Godverdomme.